google_keyboard_shortcuts.js

```diff
@@ -55,11 +57,6 @@ const initPage = () => {
   let isFirstNavigation = true;
   let resultIndex = 0;
   let results = Array.prototype.slice.call(document.querySelectorAll('h3.r a'));
-  let prevPage = document.querySelector('#pnprev');
-  if (prevPage != null)
-    results.push(prevPage);
-  let nextPage = document.querySelector('#pnnext');  
-  results.push(nextPage);

```

Comment:
I think it's still worth having the previous/next page there, because it will be easier for users to discover. For example, users may think that if they continue clicking down for the next result they will reach the next page (which is the case in some UIs). If they do this, although they won't get to the next page, they will see an easy way to get there.

Reply:
I returned this and updated it slightly, because the previous implementation could cause console errors in certain cases.